### PR TITLE
Issue #12124: Minor changes in in shell scripts

### LINKED ISSUE #12124: Minor changes in in shell scripts

### PR DESCRIPTION
**Pull request by Rahulkhinchi03**
Fixes https://github.com/checkstyle/checkstyle/issues/12124

### CODE REVIEW

File: .ci/publish-releasenotes-githubpage.sh [hunk 1]
```diff
@@ -15,10 +15,5 @@ CURRENT_VERSION=$(xmlstarlet sel -N pom=http://maven.apache.org/POM/4.0.0 \
                              -t -m pom:project -v pom:version pom.xml | sed "s/-SNAPSHOT//")
 echo CURRENT_VERSION="$CURRENT_VERSION"
```
Comment (romani) [inline .ci/publish-releasenotes-githubpage.sh:16]: why we need this variable ?

**Review by romani — CHANGES_REQUESTED** [review]
items:

File: .ci/publish-releasenotes-githubpage.sh (hunk 1 above)
Comment (romani) [reply .ci/publish-releasenotes-githubpage.sh:16]: plase use https://github.com/checkstyle/checkstyle/issues/12128 as issue reference

Comment (Rahulkhinchi03) [reply .ci/publish-releasenotes-githubpage.sh:16]: Changes Done

File: .ci/publish-releasenotes-githubpage.sh [hunk 2]
```diff
@@ -11,14 +11,5 @@ fi
 TARGET_VERSION=$1
 echo TARGET_VERSION="$TARGET_VERSION"
 
-CURRENT_VERSION=$(xmlstarlet sel -N pom=http://maven.apache.org/POM/4.0.0 \
```
Comment (romani) [inline .ci/publish-releasenotes-githubpage.sh:14]: Can we remove xmlstartlet from installation in action ?

**Review by romani — CHANGES_REQUESTED** [review]
Items

**Comment by romani** [comment]
Fix description to point to correct issue number

File: .ci/publish-releasenotes-githubpage.sh (hunk 2 above)
Comment (Rahulkhinchi03) [reply .ci/publish-releasenotes-githubpage.sh:14]: Changes Done

**Comment by Rahulkhinchi03** [comment]
already referencing to the correct issue number only.

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by romani** [comment]
Description is updated

**Pull request merged**